PatchNotes/Altibase_7.1/eng/Altibase_7_1_0_9_4_Patch_Notes.md

```diff
@@ -0,0 +1,289 @@
+# Altibase 7.1.0.9.4 Patch Notes
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+# **Table of Contents**  
+
+- [New Features](#new-features)
+    - [Improve V\$DISK\_BTREE\_HEADER and V\$MEM\_BTREE\_HEADER performance views to include index status information.](#bug-50768)
+    - [BUG-50812 cdbc (C API) support on Windows Clients.](#bug-50812)
+- [Fixed Bugs](#fixed-bugs)
+    - [BUG-50433 Improve to avoid accessing corrupted pages during Disk Page Recovery or service.](#bug-50433)
```

Comment:
service operation이 아니고,, Service phase 입니다. (알티베이스 구동단계중)

Reply:
여기 관련해서 사실 질문이 있는데요...
1. Disk Page Recovery가 정확히 어떤 동작인지..? 구동 단계 META 에서 일어나는 동작을 의미하는 건가요?
2. Service phase라는게...구동이 다 되어 운영을 하는 중인 그 때를 의미하는건지(그럼 그냥 during service해도 될것 같구요) 아니면 메타 단계에서 서비스 단계로 넘어가는 그 순간..?을 의미하는 건지...저는 사실 전자로 파악해서 쓴거긴 한데 확실하게 하는 게 필요할 거같아요ㅠㅠ

Reply:
소윤님이 이해한게 맞아요~. 전자로 파악하고 쓴거면, 나는 during service를 보고, 운영중인 상태로 해석이 되어서,, 전자로 보이게 service phase로 하자는 의견이었어요. disk page recovery 도 meta 단계에서 동작이 맞구요~

Reply:
```suggestion
    - [BUG-50433 Improve to avoid accessing corrupted pages during META or SERVICE phase.](#bug-50433)
```

이렇게 하면 어떨까 싶은데, 이것 외에 다른 것은 리뷰 사항 없을까요?